platform/android/export/export_plugin.cpp

```diff
@@ -1667,7 +1667,7 @@ void EditorExportPlatformAndroid::get_export_options(List<ExportOption> *r_optio
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, launcher_adaptive_icon_foreground_option, PROPERTY_HINT_FILE, "*.png"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, launcher_adaptive_icon_background_option, PROPERTY_HINT_FILE, "*.png"), ""));
 
-	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "graphics/32_bits_framebuffer"), true));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::INT, "graphics/depth_buffer_bits", PROPERTY_HINT_RANGE, "16,32,8"), 24));
```

Comment:
If there's only three possible values, a `PROPERTY_HINT_ENUM, "16-bit,24-bit,32-bit"` might be better than a range slider with step 8.

Reply:
Makes sense, but I'd then have a grammar/style problem _`depth_buffer_bits` is 16-bit_ doesn't sound right to me.
Other options:
- `"graphics/depth_buffer_bits", ..., "16,24,32"` (just the numbers, but maybe confusing that 0, 1, 2 map to 16, 24, 32?)
- `"graphics/depth_buffer", ..., "16-bit,24-bit,32-bit"` (since N-bit is an adjective, I think, think is like saying 'the depth buffer is a 16-bit one', but I don't quite like the name of the setting without the 'bits' part)

What do you think?

Reply:
Yeah "32-bit" vs "32 bits" can be a bit awkward, I also wondered about what's best here.

Could be `graphics/depth_buffer_bits` and `"16 bits,24 bits,32 bits"`. A bit redundant but grammatically correct.

Reply:
That's good enough. Let's settle on that. I declare the bikeshedding adjourned! 😀

Reply:
Well, last thing. Maybe being clear about what the default is may help those that have already fiddled with the settings and would like to get the default back; like this: `"16 bits,24 bits [default],32 bits"`. Sounds good?

Reply:
That could work yeah, though the more core fix would be to implement support for the revert button on export preset settings too.

Reply:
That'd be bette indeed. I'm taking note of it to maybe work on it at some point. For now, I'm just updating these PRs with the simple way.